src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/UserAcspMembership.java

```diff
@@ -42,10 +47,22 @@ public ResponseEntity<ResponseBodyPost> addAcspMember(
 
     @Override
     public ResponseEntity<AcspMembership> getAcspMembershipForAcspId(
-            @NotNull String xRequestId,
-            @Pattern(regexp = "^[a-zA-Z0-9]*$") String id
+            final String xRequestId,
+            final String id
     ) {
-        return null; // TODO(https://companieshouse.atlassian.net/browse/IDVA6-1146)
+      LOG.debug(String.format("%s: Retrieving RequestId and membership id (%s) ...", xRequestId, id));
```

Comment:
Could we be consistent with our logging?  This is difficult because we're all working on different branches and had no established pattern yet, but we could re-use the one here:

https://github.com/companieshouse/acsp-manage-users-api/blob/b56fdd894a8cbee67ac346c4392f338c5ce774dd/src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/UserAcspMembership.java#L54-L57

Or we could change that to match your new pattern of the requestId first, no strong opinions just would prefer for us to be consistent